addons/ccm-azure/ccm-azure.yaml

```diff
@@ -281,41 +277,41 @@ spec:
     spec:
       priorityClassName: system-node-critical
       serviceAccountName: cloud-node-manager
-      hostNetwork: true # required to fetch correct hostname
+      hostNetwork: true   # required to fetch correct hostname
```

Comment:
we probably should experiment with disabling this, as the node-manager get the $(NODE_NAME) as a flag and dien't need to fetch anything

Reply:
TBH, I would leave it as it is. Pretty much all other CCMs have hostNetwork enabled, and who knows is it only required for the hostname or something else as well. Even if we would try to disable it, I would leave it for a follow-up and merge this PR as it is (since this is not new and we were doing this since the beginning).